package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "braintree-angular",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "Use Braintree in your Angular app",
-  "main": "index.js",
+  "main": "dist/braintree-angular.js",
```

Comment:
I _think_ that should be ok. Looking at a couple other client-side repos it seems like that's the way to go.